src/ucp/rndv/proto_rndv.h

```diff
@@ -11,6 +11,10 @@
 
 #include <ucp/proto/proto_multi.h>
 
+/* Names of rendezvous acknowledgment messages */
```

Comment:
[minor] add one more empty line? (e.g. like in [src/ucp/rndv/rndv_get.c](https://github.com/openucx/ucx/pull/8308/files#diff-ac996cf0c5ee1c54da915a335063af78605d944b4634970bfa3247d3cf3713cf))